2012-06-06  Aurelien Jarno  <aurelien@aurel32.net>

	* locales/es_MX (LC_MONETARY): Set mon_thousands_sep to space
	(<U0020>). (LC_NUMERIC): Set decimal_point to dot (<U002E>)
	and thousands_sep to space (<U0020>).

---
 localedata/locales/es_MX |    6 ++++--
 1 file changed, 4 insertions(+), 2 deletion(-)

--- a/localedata/locales/es_MX
+++ b/localedata/locales/es_MX
@@ -63,7 +63,7 @@ LC_MONETARY
 int_curr_symbol      "<U004D><U0058><U004E><U0020>"
 currency_symbol      "<U0024>"
 mon_decimal_point    "<U002E>"
-mon_thousands_sep    "<U002C>"
+mon_thousands_sep    "<U0020>"
 mon_grouping         3;3
 positive_sign        ""
 negative_sign        "<U002D>"
@@ -78,7 +78,9 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-copy "es_ES"
+decimal_point        "<U002E>"
+thousands_sep        "<U0020>"
+grouping             3;3
 END LC_NUMERIC
 
 LC_TIME
